src/xo-app/navbar/index.js

```diff
@@ -0,0 +1,41 @@
+import Icon from 'icon'
+import React, { Component } from 'react'
+import {
+  connectStore,
+  propTypes
+} from 'utils'
+
+@connectStore([
+  'user'
+])
+@propTypes({
+  selectLang: propTypes.func.isRequired
+})
+export default class Navbar extends Component {
+  handleSelectLang (event) {
+    // FIXME: find a way to reach selectLang to set the app language
+    this.props.selectLang(event.target.value)
```

Comment:
@julien-f I know I can't access this directly, but I don't know how to proceed...